nmostesting/suites/IS0401Test.py

```diff
@@ -1515,8 +1515,8 @@ def test_23(self, test):
             return test.PASS()
         else:
             return test.OPTIONAL("No BCP-002-01 groups were identified in Sender or Receiver tags",
-                                 "https://amwa-tv.github.io/nmos-grouping/branches/master"
-                                 "/best-practice-natural-grouping.html")
+                                 "https://specs.amwa.tv/bcp-002-01/branches/{}"

```

Comment:
I think this needs a _specific_ branch, not the `{}` placeholder, because we don't have an `api["spec_branch"]` for BCP-002-01.

I assume we're going to make a BCP-002-01 v1.0.0 release, and make a v1.0.x branch, so it should probably be `v1.0.x` ultimately, but for now maybe...
```suggestion
                                 "https://specs.amwa.tv/bcp-002-01/branches/v1.0-dev"
```

Reply:
Ok that makes sense.  We've needed the v1.0.0 release for some time now, so makes sense to do that and change to that rather than v1.0-dev.